modules/stochastic_tools/src/transfers/SamplerPostprocessorTransfer.C

```diff
@@ -13,14 +13,16 @@
 #include "SamplerTransientMultiApp.h"
 #include "SamplerReceiver.h"
 #include "StochasticResults.h"
+#include "Sampler.h"
 
 registerMooseObject("StochasticToolsApp", SamplerPostprocessorTransfer);
 
-template <>
+defineLegacyParams(SamplerPostprocessorTransfer);
+
 InputParameters
-validParams<SamplerPostprocessorTransfer>()
+SamplerPostprocessorTransfer::validParams()
 {
-  InputParameters params = validParams<StochasticToolsTransfer>();
+  InputParameters params = StochasticToolsTransfer::validParams();
   params.addClassDescription("Transfers data from Postprocessors on the sub-application to a "
                              "VectorPostprocessor on the master application.");
   params.addRequiredParam<PostprocessorName>(
```

Comment:
"The name of the Postprocessors on the sub-app to transfer from/to." -> I thought we can only do one-way transfer, so maybe change to "The name of the Postprocessors on the sub-app to transfer from." ? 

Same for the 'vector_postprocessor' description.

Maybe we should give a better name to "postprocessor" and "vector_postprocessor". It is a bit hard to understand their difference. 'from_postprossor' and 'to_vector_postprocessor'?